### PR TITLE
Added Hello World in Erlang Article

### LINKED ISSUE #321: Add Article for Hello World in Erlang
See #320 for changes. Looking for author!

### PR DESCRIPTION
**Pull request by nickkeers**
Congrats on taking the first step to contributing to the Sample Programs repository maintained by [The Renegade Coder][1]! For simplicity, please make sure that your pull request includes one and only one contribution.

## Complete the Applicable Sections Below

Find which section best describes your pull request and make sure you fill it out. To start, let us know which issue you've fixed.

- [x] I fixed #321 

### Code Snippets

- [ ] I named the pull request using `Added/Updated <Sample Program> in <Language>` format
- [ ] I created/updated the language README
  - [ ] I added the sample program name to the README
  - [ ] I added fun facts (i.e. debut, developer, typing, etc.)
  - [ ] I added reference link(s) to the README
  - [ ] I added solution citations when necessary (see [plagiarism][2])

### Documentation

- [x] I named the pull request using `Added/Updated <Sample Program> in <Language> Article` format
- [x] I followed the applicable article template
  - [ ] I added/updated a [language article][4] (i.e. The Python Programming Language)
  - [x] I added/updated a [code snippet article][3] (i.e. Hello World in Perl)
  - [ ] I added/updated a [project article][5] (i.e. Fizz Buzz in Every Language)

### Testing

- [ ] I names the pull request using `Added/Updated <Language>/<Project> Testing` format
- [ ] I followed the testinfo template, if applicable

## Notes

Added the Hello world in Erlang article, simplified the Hello world in Erlang code snippet to use the simpler form of `io:format`. Added myself as an author in `authors.yml`, added and linked to a featured image in the `assets/images` folder

[1]: https://therenegadecoder.com/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#plagiarism
[3]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/CODE_ARTICLE_TEMPLATE.md
[4]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/LANGUAGE_ARTICLE_TEMPLATE.md
[5]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/PROJECT_ARTICLE_TEMPLATE.md

### CODE REVIEW

File: docs/projects/hello-world/_posts/2019-09-04-erlang.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: Hello world in Erlang
```
Comment (jrg94) [inline docs/projects/hello-world/_posts/2019-09-04-erlang.md]:
```suggestion
title: Hello World in Erlang
```

File: docs/projects/hello-world/_posts/2019-09-04-erlang.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+layout: default
+last-modified: 2019-09-04
```
Comment (jrg94) [inline docs/projects/hello-world/_posts/2019-09-04-erlang.md]: Make sure you update the file name as well.

```suggestion
last-modified: 2019-10-04
```

File: docs/projects/hello-world/_posts/2019-09-04-erlang.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+featured-image: hello-world-in-erlang-featured-image.JPEG
+tags: [erlang, hello-world]
+authors:
+  - the_renegade_coder
```
Comment (jrg94) [inline docs/projects/hello-world/_posts/2019-09-04-erlang.md]: Since you're the primary author, you can pull me off this (or at least swap the order). 😃

**Comment by nickkeers** [comment]
Resolved the latest QA issues in the newest commit!

**Comment by jrg94** [comment]
I believe this fulfills the 100 article milestone. Woo!

**Comment by nickkeers** [comment]
> 
> 
> I believe this fulfills the 100 article milestone. Woo!

Awesome news :) may tackle a few other issues too!

**Pull request merged**